LSICORR/iocBoot/iocLSICORR-IOC-01/config.xml

```diff
@@ -4,6 +4,9 @@
   <ioc_desc>LSI Correlator</ioc_desc>
   <ioc_details>LSI Correlator component of SANS-DLS</ioc_details>
   <macros>
+    <macro name="FILEPATH" pattern="^.*$" description="Directory in which to save data files." hasDefault="UNKNOWN"/>
+    <macro name="ADDR" pattern="^.*$" description="IP address to connect to." hasDefault="UNKNOWN" />
```

Comment:
Use either YES or NO here, I've added a note to https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches#8-macros-and-details about it.